dask/tests/test_dot.py

```diff
@@ -271,3 +272,21 @@ def f(x, y=True):
     label = task_label(x.dask[x.key])
     assert "f" in label
     assert "apply" not in label
+
+
+def test_immutable_attributes():
+    def inc(x): return x + 1
+    dsk = {"a": (inc, 1), "b": (inc, 2), "c": (add , "a", "b")}
+    attrs_func = {"a": {}}
+    attrs_data = {"b": {}}
+    attrs_func_test = copy.deepcopy(attrs_func)
+    attrs_data_test = copy.deepcopy(attrs_data)
+
+    to_graphviz(
+        dsk,
+        function_attributes=attrs_func,
+        datat_attributes=attrs_data,
```

Comment:
There is a typo in here
```suggestion
        data_attributes=attrs_data,
```

Reply:
Also don't you want to test that the output of to_graphviz has the values that you'd expect?